src/main/kotlin/com/github/mitchellolsthoorn/testgenie/settings/TestGenieSettingsState.kt

```diff
@@ -3,5 +3,36 @@ package com.github.mitchellolsthoorn.testgenie.settings
 /**
  * This class is the actual data class that stores the values of the Settings entries.
  */
-data class TestGenieSettingsState
-    constructor(var globalTimeout: String = "60", var showCoverage: Boolean = false)
\ No newline at end of file
+data class TestGenieSettingsState(
+        var showCoverage: Boolean = false,
+
+        var sandbox : Boolean = true,
+        var assertions : Boolean = true,
+        var seed : String = "",
+        var algorithm: ContentDigestAlgorithm = ContentDigestAlgorithm.RANDOM_SEARCH,
+        var configurationId : String = "",
```

Comment:
Why everything is set to true? Are all these the default values in EvoSuite?

Reply:
All criteria set here to `true` are by default set to `true` in EvoSuite. We tried copying EvoSuite CLI's defaults